src/components/Modal/VaultStrategyModal/managedVaults/ManagedVaultsStrategyModalApr.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from "react"
+
+import { VaultStrategyModalDefinitionList } from "@/components/Modal/VaultStrategyModal"
+
+export const ManagedVaultsStrategyModalApr: FC = () => {
+  const breakdownApr = {
+    isLoading: false,
+    data: {
+      GMXApr: 0.1,
+      ETHApr: 0.23,
+      totalApr: 0.33,
+    },
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const data = breakdownApr.data as any
+
+  // eslint-disable-next-line no-sparse-arrays
```

Comment:
I don't think this is needed?

Reply:
atm as I mocked the data above I need it, otherwise there is an exception when I check all fields below